packages/platforms/android-player/android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.getcapacitor.myapp;
```

Comment:
Is this file needed? (androidTest, etc)

Reply:
No. It was generated by capacitor and I'll remove it